### ISSUE TITLE
Football datasets

### ISSUE THREAD

**Issue by rufuspollock** [datahubio/awesome-data#1]
It would be great to get a list of "awesome" football related datasets.

@geraldb would you be up for helping curate this as i know you are something of an expert. What we'd be looking for is just a list of great datasets in this area.

## Research

* Great set of data here https://github.com/openfootball
* Some existing datasets here https://old.datahub.io/dataset?tags=football (many of links seem dead though)

**Comment by geraldb** [comment]
@rufuspollock  Thanks for the consideration. Unfortunately I try to maintain over [500+ projects on github](https://github.com/yorobot/backup/blob/master/SUMMARY.md) so I'm already strechted trying to update those :-).  

The reality is there's close to zero (structured) open football data.  

The "definite"  for collecting sources (I know of) is ["Guide to Football/Soccer data and APIs"](https://www.jokecamp.com/blog/guide-to-football-and-soccer-data-and-apis/) last updated on June 21 2017by Joe Kampschmidt (@jokecamp) 

I started an [Awesome Football page @ Planet Open Data](https://github.com/planetopendata/awesome-football) for collecting datasets some time (years) ago - unfortunately - not really updated for some time now.

Greetings from Vienna. Happy new year! Prosit 2018!  And as always thanks for your great inspiring open   / frictionless data initiatives.  Keep it up.

**Comment by rufuspollock** [comment]
@geraldb totally get the stretched part 😄 and really appreciate the detailed comment. Happy new year to you 🎄 🍷 

Would you mind if we re-used some the material from the Awesome Football page here (with appropriate credit of course)?

> The best definite guide for collecting sources (I know of) is "Guide to Football/Soccer data and APIs" last updated on June 21 2017by (the great ) Joe Kampschmidt (@jokecamp) - Big thanks!

@geraldb thanks for pointing this out - it looks great. @jokecamp this is a great resource and if you had any comments they'd be warmly welcomed.

**Comment by geraldb** [comment]
> Would you mind if we re-used some the material from the Awesome Football page here (with appropriate credit of course)?

Of course, it's an honor - please, (re)use from the page as much material as you please - that what it's here. It's public domain - no credit needed.

Awesome idea ;-) the awesome page on the new datahub.io site. Keep it up. Prosit 2018!
